types/src/validator_txn.rs

```diff
@@ -11,6 +11,7 @@ pub enum ValidatorTransaction {
     DummyTopic1(DummyValidatorTransaction),
     DKGResult(DKGTranscript),
     DummyTopic2(DummyValidatorTransaction),
+    ObservedJWKUpdate(jwks::QuorumCertifiedUpdate),
```

Comment:
we should reorganize these, having dummy, real, dummy, real is weird

Reply:
fixed, dummy has gone